docs/source/roadmaps/june-2024.md

```diff
@@ -0,0 +1,144 @@
+# June 2024
+
+One of our main goals in the team is to provide fully integrated, automated analysis tools for systems neuroscience 
+data. As a researcher, this would mean providing only a set of configurations before running the analysis for an entire 
+dataset, with results and quality metrics output for checking. This will avoid researchers needing to write 
+custom preprocessing and analysis scripts that often duplicate existing efforts, and allows researchers to work 
+together on community tools.
+
+This is a long-term goal, and we are working across multiple themes towards achieving this. We are ensuring [data is 
+organised in a standard way](data_management), a critical step towards automation. Currently, we are focused on 
+building pipelines for [behaviour](behaviour), [electrophysiology](electrophysiology) and [anatomy](anatomy) data that 
+can ingest data from various acquisition systems into a common interface. These are critical steps towards the 
+longer-term goal of multimodal integration. The detailed roadmap is split into these themes below.
+
+(data_management)=
+## Data management
+**Current status**
+
+We have developed [NeuroBlueprint](https://neuroblueprint.neuroinformatics.dev/), a simple, standard data folder 
+specification for (systems) neuroscience and [datashuttle](https://datashuttle.neuroinformatics.dev/), to automate the 
+creation, validation and transfer of NeuroBlueprint projects. For more details on this specific project, see the 
+[datashuttle roadmap]().
```

Comment:
@JoeZiminski I saw https://github.com/neuroinformatics-unit/NeuroBlueprint/pull/58 has been merged. Is there going to be a datashuttle roadmap, or should the neuroblueprint one be linked here instead (or both)?

Reply:
Hey Adam the datashuttle roadmap was folded in [the community page PR](https://github.com/neuroinformatics-unit/datashuttle/pull/385), I think it is good to go @niksirbi may want to take a look. The link to the NB page should be added, maybe there can be a NB version bump to publish the new docs, then link to NB from datashuttle PR, then merge  the datashuttle roadmap PR. Then both can be linked to here.

Reply:
Sounds good. Are you happy to update this PR when this has all happened?

(just trying to make sure the may roadmap doesn't become the July roadmap!).

Reply:
Thanks @JoeZiminski!